### PR TITLE
[RFC] vim-patch:6921742,  vim-patch:37c2403 (runtime updates)

### PR DESCRIPTION
**Pull request by fwalch**
No manual changes, only ran `patch -p1 < vim-6921742.diff`, with following output:

```
patching file runtime/doc/eval.txt
Hunk #3 succeeded at 2632 (offset 10 lines).
Hunk #4 succeeded at 2923 (offset 10 lines).
Hunk #5 succeeded at 5559 (offset 68 lines).
Hunk #6 succeeded at 5708 (offset 67 lines).
can't find file to patch at input line 63
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|diff --git a/runtime/doc/os_vms.txt b/runtime/doc/os_vms.txt
|--- a/runtime/doc/os_vms.txt
|+++ b/runtime/doc/os_vms.txt
--------------------------
File to patch: 
Skip this patch? [y] 
Skipping patch.
2 out of 2 hunks ignored
patching file runtime/doc/os_win32.txt
Hunk #2 FAILED at 334.
1 out of 2 hunks FAILED -- saving rejects to file runtime/doc/os_win32.txt.rej
patching file runtime/doc/pattern.txt
patching file runtime/doc/syntax.txt
patching file runtime/doc/tags
Hunk #1 FAILED at 4329.
1 out of 1 hunk FAILED -- saving rejects to file runtime/doc/tags.rej
patching file runtime/doc/todo.txt
Hunk #7 succeeded at 240 (offset -3 lines).
Hunk #8 succeeded at 1441 (offset -10 lines).
patching file runtime/doc/usr_04.txt
can't find file to patch at input line 281
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|diff --git a/runtime/doc/version6.txt b/runtime/doc/version6.txt
|--- a/runtime/doc/version6.txt
|+++ b/runtime/doc/version6.txt
--------------------------
File to patch: 
Skip this patch? [y] 
Skipping patch.
2 out of 2 hunks ignored
can't find file to patch at input line 299
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|diff --git a/runtime/doc/version7.txt b/runtime/doc/version7.txt
|--- a/runtime/doc/version7.txt
|+++ b/runtime/doc/version7.txt
--------------------------
File to patch: 
Skip this patch? [y] 
Skipping patch.
6 out of 6 hunks ignored
patching file runtime/filetype.vim
Hunk #2 succeeded at 526 (offset -4 lines).
patching file runtime/ftplugin/dockerfile.vim
patching file runtime/indent/json.vim
patching file runtime/scripts.vim
Hunk #2 succeeded at 314 (offset -5 lines).
patching file runtime/syntax/asciidoc.vim
patching file runtime/syntax/dockerfile.vim
```

The rejected change to the `tags` file is irrelevant, as that file is not in Neovim. The rejected `win32` hunk is already in Neovim. Contents of `runtime/doc/os_win32.txt.rej`:

``` diff
--- runtime/doc/os_win32.txt
+++ runtime/doc/os_win32.txt
@@ -334,7 +334,7 @@
         :!start /min ctags -R .
 <  When it has finished you should see file named "tags" in your current
    directory.  You should notice the window title blinking on your taskbar.
-   This is more noticable for commands that take longer.
+   This is more noticeable for commands that take longer.
    Now delete the "tags" file and run this command: >
         :!start /b ctags -R .
 <  You should have the same "tags" file, but this time there will be no
```

### CODE REVIEW

**Comment by fwalch** [comment]
Travis fails on helptags generation because a reference to (not yet ported) E888 is added by this PR.

**Comment by fwalch** [comment]
Travis fails because this patch contains an error: a `*E888*` should be `|E888|` (in `patterns.txt`). This has been fixed in Vim changeset [37c24033b2601a4a0953970a0ec85d5537101915](https://code.google.com/p/vim/source/detail?r=37c24033b2601a4a0953970a0ec85d5537101915).

File: runtime/doc/cmdline.txt [hunk 1]
```diff
@@ -56,7 +56,7 @@ history tables:
 These are completely separate.  Each history can only be accessed when
 entering the same type of line.
 Use the 'history' option to set the number of lines that are remembered
-(default: 20).
+(default: 50).
```
Comment (fwalch) [inline runtime/doc/cmdline.txt]: This is consistent with the default mentioned in `runtime/doc/options.txt`. However, Neovim's default still seems to be the previous value (`20`). Vim 7.4.537 already has the default of `50`.

Comment (fwalch) [inline runtime/doc/cmdline.txt]: Added a commit to address that.

**Comment by justinmk** [comment]
LGTM, thanks!

**Pull request merged**